extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts

```diff
@@ -187,8 +187,8 @@ export class NumericInputValidationService {
 
   parseValue(viewValue: string): number {
     if (viewValue) {
-      viewValue = viewValue.trim();
-      return parseFloat(viewValue);
+      // Remove commas and leading/trailing spaces before parsing.
+      return parseFloat(viewValue.replace(/\,|^\s+|\s+$/g, ''));
```

Comment:
Can you maybe still use trim and then only replace the `,`, I think it will be a bit more readable.

Reply:
Done.

Reply:
The reason I had removed trim is because it seemed like 2 passes will be done on the string. One time to remove spaces and the second time to replace ','. But I agree that readability is more important and therefore have changed it.